kde-apps/kate-common/kate-common-9999.ebuild

```diff
@@ -0,0 +1,33 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+ECM_HANDBOOK="true"
+KDE_ORG_NAME="${PN/-common/}"
+inherit ecm-common gear.kde.org
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS=""
+
+RDEPEND="
+	!<kde-apps/kate-24.07.90-r1
+	!<kde-apps/kate-addons-24.07.90-r1
+	!<kde-apps/kate-lib-24.07.90-r1
+"
+
+src_prepare() {
+	ecm-common_src_prepare
+
+	# delete colliding kwrite translations
+	find po -type f -name "*po" -and -name "kwrite*" -delete || die
```

Comment:
Why doesn't ecm_punt_po_install work here?

Reply:
In this case we pretend kwrite is still like an independent package. Otoh, it itself depends on kate-lib anyway, so we might as well do the new standard treatment...